src/haggadah/views.cljs

```diff
@@ -122,40 +122,101 @@
 
+(defn icon
+  "Pre: takes two Haggadah ids
+  Post: returns a filled circle icon if the ids match. Returns a unfilled circle icon otherwise"
+  [id-1 id-2]
+  (if (= id-1 id-2)
+    [:i.fa-circle.fas]
+    [:i.fa-circle.far]))
+
+(defn all-haggadot
+  "Pre: takes a collection of Haggadot and a selected Haggadah
+  Post: returns a hiccup representation of a collection of Haggadot"
+  [haggadot]
+   (let [haggadah-picked @(re-frame/subscribe [::subs/selected-haggadah])]
+     [:div.dropdown-content
+      (for [{:keys [title id]} haggadot :when id]
+        ^{:key id}
+        [:a.dropdown-item.seder-option
+         {:on-click #(re-frame/dispatch [::events/selected-haggadah id])
+          :data-id id}
+         title
+         [icon haggadah-picked id]])]))
+
+(defn form-content
+  "Pre: takes an id for a form field
+  Post: returns the text of the field if it exists, nil otherwise"
+  [id]
+  (-> (.getElementById js/document id)
+      (.-value)))
+
+
+(defn seder-popup
+  []
+  (let [id @(re-frame/subscribe [::subs/seder-modal])
+        active (when id "is-active")]
+    [:div.modal {:class active}
+     [:div.modal-background]
+     [:div.modal-content
+      [:form.box
+       [:div.text-centered.pb-2
+        "Please enter the title of the Seder"]
+       [:div.field
+        [:input#seder-title.input {:type "email" :defaultValue "The title of your Seder"}]]
+       [:div.field.is-grouped.is-grouped-left 
+        [:div.control 
+         [:a.button.is-small.button  {:on-click #(re-frame/dispatch [::events/hide-seder-modal])} "Cancel"]]
+        [:div.control 
+         [:a.button.is-small {:class (styles/submit-button) :on-click  #(re-frame/dispatch [::events/create-seder id
+                                                                                            (form-content "seder-title")])
+                              :data-testid :submit} "Create"]]]]]
+     [:button.modal-close.is-large]]))
+
+
 (defn dashboard-panel
   []
   [:div.page 
    [:div.container.is-large.hero.is-flex
     [:div.hero-body.pt-6
-     [:div.pt-24.column
-      (let [name (re-frame/subscribe [::subs/name])]
+     (let [haggadot @(re-frame/subscribe [::subs/haggadot])
+           sedarim @(re-frame/subscribe [::subs/sedarim])]
+       [:div.pt-24.column
+        (let [name (re-frame/subscribe [::subs/name])]
+          [:div
+           [:h1.text-center.is-size-4 {:data-testid :user}
+            (str "Hello " @name ". Welcome. To make a new Haggadah, click the button to your right. To share and edit your existing Haggadah, look at your Haggadot below ")]])
```

Comment:
Perhaps for the future the name should be on the menu (on the right is very common)... with a logout option

Reply:
I'll make an issue for moving the name to the menu.